build.sbt

```diff
@@ -33,7 +33,7 @@ lazy val root = Project("kettle-test-framework", file("."))
       )
     ),
     scalacOptions ++= Seq(
-      "-target:jvm-1.8",
+      "-release:8",
```

Comment:
I think this should be `"-release", "8"` - see my comments on https://github.com/nationalarchives/tna-cat/pull/326

Reply:
Done